index.js

```diff
@@ -3,3 +3,16 @@ const btn = document.querySelector('button');
 btn.onclick = () => {
     alert('This Button Works!')
 }
+
+window.addEventListener("load", (event) => {
+    console.log("page is fully loaded")
+    fetch("http://api.weatherapi.com/v1/current.json?key=f21c22a7558242f0ac7151418240506&q=37.91822293084342, 21.745074031336777&aqi=no").then(resp=>resp.json()).then(data=>{
+        console.log(data)
+        const current = data.current.temp_c
+        console.log(current)
+        document.getElementById("weatherloc").textContent=current+" C!";
```

Comment:
Something fun to learn about javascript, proper string concatenation. 

Concatenation using the `+` does not enforce the type, meaning in this case:

If our api here where to return a value other than a string for `temp_c`, say a segment of nefarious code, then our code could be vulnerable here.

Instead, let's use the tilde `` ` ``  instead of quotes for a string, then inside that string we can use `${   }` to switch back to javascript and insert our variable.

```suggestion
        document.getElementById("weatherloc").textContent=`${current} C!`;
```

What this does is ensure that current will be compiled as a string, preventing nefarious code from injecting hacks.